plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -163,6 +164,27 @@ const AddressForm = ( {
 					);
 				}
 
+				if ( field.type === 'select' ) {
+					if ( typeof field.options === 'undefined' ) {
+						return null;
+					}
+					return (
```

Comment:
What are your thoughts on adding a line break before the return statement for readability reasons?

```suggestion
					if ( typeof field.options === 'undefined' ) {
						return null;
					}
					
					return (
```